src/bci_build/package/postfix.py

```diff
@@ -41,6 +43,7 @@
         custom_description="Postfix container is fast and secure mail server, {based_on_container}.",
         os_version=os_version,
         is_latest=os_version in CAN_BE_LATEST_OS_VERSION and os_version.is_tumbleweed,
+        from_target_image=f"{_build_tag_prefix(os_version)}/bci-micro:{OsContainer.version_to_container_os_version(os_version)}",
```

Comment:
you can now use `generate_from_image`

Reply:
actually not, because `from_target_image` behaves differently than `from_image`. I guess I could align them.